.github/workflows/build-test-osx-arm64.yaml

```diff
@@ -13,13 +13,24 @@ on:
         required: true
         type: boolean
         default: true
+      wheel-artifact-name:
+        description: "The name under which to store the wheel artifact"
+        type: string
+        default: osx-arm64-wheel
```

Comment:
Is there anywhere where we set the wheel-artifact-name for this workflow to something other than osx-arm64-wheel?
If not, then what's the point of introducing this extra intermediate constant that looks like a variable but is not?

Reply:
In release.yml, there is one other place this is referenced in the `upload-wheels` job. Unfortunately, we can't set an env var in release.yml to hold this value, but this will likely be helpful with some follow up work I have on deck to factorize the OSX workflows.

Reply:
if it's referenced but never set to a different value, it means it's a constant and we should just inline its content instead of introducing those intermediate name that complicate things IMHO.

Reply:
Ok, I can remove that bit then. thanks!

Reply:
@aryx swapped that around to make it such that the wheel artifact names are outputs which can be defined once and referenced, not inputs - lmk if that's acceptable. thanks!